src/wired.d.ts

```diff
@@ -1,27 +1,27 @@
 declare namespace JSX {
```

Comment:
why did you touch this file? please revert any changes here

Reply:
Without this change I can't add attributes like "checked, disabled..." in to the wired-elements. Typescript give me an issue, and I haven't another solution. I did the change in two elements anb add one element "wired-item"

Reply:
done